<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Apollo</class>
 <widget class="QDialog" name="Apollo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>879</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QStackedWidget" name="AstackedWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>175</number>
         </property>
         <item>
          <widget class="QPushButton" name="pushButton_6">
           <property name="styleSheet">
            <string notr="true">background-color: blue;  
                border: 2px solid white;  
                border-radius: 12px;       
                padding: 10px 24px;    
                font-size: 20px;             
                font-weight: bold;        
                text-align: center;   
                width: 200px;                
                height: 50px;       </string>
           </property>
           <property name="text">
            <string>Back</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_7">
           <property name="styleSheet">
            <string notr="true">background-color: blue;  
                border: 2px solid white;  
                border-radius: 12px;       
                padding: 10px 24px;    
                font-size: 20px;             
                font-weight: bold;        
                text-align: center;   
                width: 200px;                
                height: 50px;       </string>
           </property>
           <property name="text">
            <string>Quit Game</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>175</number>
         </property>
         <item>
          <widget class="QPushButton" name="acceptBtn">
           <property name="styleSheet">
            <string notr="true">background-color: blue;  
                border: 2px solid white;  
                border-radius: 12px;       
                padding: 10px 24px;    
                font-size: 20px;             
                font-weight: bold;        
                text-align: center;   
                width: 200px;                
                height: 50px;           
              </string>
           </property>
           <property name="text">
            <string>Accept Mission</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="scoreBtn">
           <property name="styleSheet">
            <string notr="true">background-color: blue;  
                border: 2px solid white;  
                border-radius: 12px;       
                padding: 10px 24px;    
                font-size: 20px;             
                font-weight: bold;        
                text-align: center;   
                width: 200px;                
                height: 50px;       </string>
           </property>
           <property name="text">
            <string>View Scores</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="1">
        <widget class="QPlainTextEdit" name="plainTextEdit">
         <property name="maximumSize">
          <size>
           <width>800</width>
           <height>200</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: white;  
                color: black;  
                border-radius: 10px;   
				border: solid blue ;   
                padding: 10px;            
                font-size: 14px;         
                font-family: Arial;
				border-color: #45a049; </string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="plainText">
          <string>I

</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Policy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Policy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>50</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Policy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Policy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>80</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="2">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Policy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>300</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Select a Level</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QPushButton" name="levelUnoBtn">
                <property name="maximumSize">
                 <size>
                  <width>200</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: blue;  
                border: 2px solid white;  
                border-radius: 12px;       
                padding: 10px 24px;    
                font-size: 20px;             
                font-weight: bold;        
                text-align: center;   
                width: 200px;                
                height: 50px;       </string>
                </property>
                <property name="text">
                 <string>Level Uno</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="levelDosBtn">
                <property name="maximumSize">
                 <size>
                  <width>200</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: blue;  
                border: 2px solid white;  
                border-radius: 12px;       
                padding: 10px 24px;    
                font-size: 20px;             
                font-weight: bold;        
                text-align: center;   
                width: 200px;                
                height: 50px;       </string>
                </property>
                <property name="text">
                 <string>Level Dos</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="levelTresBtn">
                <property name="maximumSize">
                 <size>
                  <width>200</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: blue;  
                border: 2px solid white;  
                border-radius: 12px;       
                padding: 10px 24px;    
                font-size: 20px;             
                font-weight: bold;        
                text-align: center;   
                width: 200px;                
                height: 50px;       </string>
                </property>
                <property name="text">
                 <string>Level Tres</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="homeBtn">
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: blue;  
                border: 2px solid white;  
                border-radius: 12px;       
                padding: 10px 24px;    
                font-size: 20px;             
                font-weight: bold;        
                text-align: center;   
                width: 200px;                
                height: 50px;       </string>
           </property>
           <property name="text">
            <string>Go to Home</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="1">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Policy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>50</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Policy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>300</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Policy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>50</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_3"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
